tests/functional/tests/rgw/test_rgws.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+class TestRGWs(object):
+
+    @pytest.mark.no_docker
+    def test_ceph_rgw_package_is_installed(self, node, Package):
+        # ceph-radosgw is the package name on centos, radosgw name on ubuntu
+        assert (Package("ceph-radosgw").is_installed or Package("radosgw").is_installed)
```

Comment:
@alimaredia Can you try splitting it into two asserts? It looks like the check for ``ceph-radosgw``is failing and the second one never runs. You might need to take a different approach here.